### ISSUE TITLE
who can be a moderator

### ISSUE THREAD

**Issue by olympusMonspeacebeuponhim** [Wookey-Technologies/magna-carta#18]
this is a discussion of setting who can be a moderator.  i have suggested changing the policy  with two employees and two users. previously there were only 3 moderators so the number is good. This is different from how it is written right now. where only sansar can say who are a moderator. heres the full proposal i made 

Appealing to authority is a logical fallacy. The active users elects, at will, people to enforce Sansarian law, but the law itself is yours, and the actions taken are public. If you find the authority being unjust, review their action and ask for them to be educated. Nobody is above you, except law, logic, and reason, and the kindness of our platform.

2.10.1 Who is a moderator
A moderator is 2 employees and 2 active users voted on by verified active users with one vote through sansar official discord. Known and unknown alts voting is not allowed. Sansar must provide a list of staff members that don't mind moderating.The list has to be long enough to allow for a choice 5 at minimum to vote on. A helper is not a moderator

2.10.2 Requirements and Terms of moderator
serve a 3 month term(subject to change if the community feels they are good at moderating)

Sansar Users that are voteda moderator if they cannot fulfill there moderating duty for reason of taking break from sansar or quitting they must make notification they do not want the job anymore and replaced.

2.10.3 Good Moderation Practices
Moderator should keep the neutral ground balance, and act according to the policies. If needing to act alone every moderation must be done with proper documentation and careful attention. Notify the other moderators immediately. In general don't intimidate, dont bird Dog(watching over every user waiting for them to need moderation). Let the flow and peoples fun always be the priority. You are a User/Staff Member first and moderator second

Punishment from moderation does not have to go the extreme of a ban. It could be a first a verbal conversation(has to be documented and public). Communications discord private message, email, sansar private message, and possibly needing to be video recorded. User needs to know he is being moderated in these communications. User cannot abuse moderator because they are doing official moderation duty. Accused user should defend their actions during moderation from communication. Moderation does not have to lead to temporary or permanent ban. Just a conversation on accusation

**Comment by 3MUl0R** [comment]
I think the content of section 2.10.3 Good Moderation Practices is very useful and some form of it should probably be a part of the enforcement section.

**Comment by olympusMonspeacebeuponhim** [comment]
You believe sansar solely should say who a moderator is?? Some of us came up with a need for balance.  It wasn't my idea alone.  Some users will be moderators and I personally never liked the helper system. I actually came up with the idea of "ambassadors" which became helpers.  But they were never the moderators in my suggestion.  They were people that frequented sansar that were nice people and wouldn't mind being  helpful to newbies. Comets, helpers were always sansar  brown noses and looked at by some as that.  It was a social title.  This way is objective and democratic.

**Comment by 3MUl0R** [comment]
`You believe Sansar solely should say who a moderator is??  `  
No, I didn't suggest that.  However, now that you mention it I do think there would be a lot of issues we (the community) should consider.  for example:
- Voting participation
  - There could easily be very low participation since everyone has things to do in their personal lives
  - How do we decide what constitutes a quorum?
    - If you only have 20 out of an active user base of 2000 show up is it legitimate to hold a vote?
- Removing bad actors
  - How would we deal with people who were elected to the moderator role but turned out to have issues of their own to deal with?
  - What if the elected moderators failed to show up regularly?
  - The list could go on and on of reasons why they may need to be replaced 
  - Sansar would very likely need to retain the ability to remove any elected community moderator 

Personal opinion: A fledgling meta verse community is not ready for such processes to be put in place.  They are likely to cause more harm than good for the foreseeable future.  It's better to have a say in the guidelines and rules we are all judged by, and since we have the ability to challenge any decisions it matters much less who is performing the moderation.

**Comment by olympusMonspeacebeuponhim** [comment]
I don't like the helpers being moderators.  So if it's solely  sansar to choose moderators they can put helpers as moderators.  Most the helpers hadn't logged in in a long time. 

The community should consider voting as something that is important.  As in US politics you don't vote you can't complain.  Voting is what makes you a citizen.  In the US you had to be male, white, over the age of 18, and a land owner.  That makes you a citizen.

If a moderator is a bad actor they  themselve should resign knowing they aren't cut out for job. That could be educated to moderators.  Recognize yourself that the job isn't for you

**Comment by olympusMonspeacebeuponhim** [comment]
Good point on quorum. Does nick have any instances in real life happen and maybe an example how a solution to someone feeling wronged won over 2000 people
